tools/ci/check_for_python_lint.sh

```diff
@@ -20,32 +20,24 @@
 HOME=${1:-"."}
 FOUND_ISSUE=-1
 
-files=`git diff --name-only origin/master --diff-filter=b | egrep .py$ | \
-  grep -v /ext-py/ | \
-  grep -v /ext-py3/ | \
-  grep -v wsgiserver.py | \
-  grep -v /migrations/ | \
-  grep -v apps/oozie/src/oozie/tests.py | \
-  grep -v tools/ops/ | \
-  grep -v apps/beeswax/gen-py/ | \
-  grep -v /org_migrations/`
+files=`git diff --name-only origin/master --diff-filter=b | egrep .py$` 
```

Comment:
Why is the list of files to lint different now?

Reply:
It is not different. Earlier we were removing these files if they were present in the staged changes before sending with the pylint command.

Now we are making the command clean for ruff by sending only the files containing the staged changes (without any exclusions). And the actual exclusion is happening in the ruff config where we are forcibly excluding all such files. 

This will also help keep this exclusion in one place and not duplicate it when we expand the ruff usage by bringing in format and auto-fix features.